stl/inc/xutility

```diff
@@ -3258,13 +3261,26 @@ namespace ranges {
             return _RANGES begin(_Self)[_Idx];
         }
     };
-    // clang-format on
 
-    // ENUM ranges::subrange_kind
+    // ENUM CLASS ranges::subrange_kind
     enum class subrange_kind : bool { unsized, sized };
+    // clang-format on
+} // namespace ranges
 
-    // CLASS TEMPLATE ranges::subrange
+// These declarations must be visible to unqualified name lookup for _STD get in _Pair_like below, even if

```

Comment:
But `_STD get` is `::std::get` which isn't unqualified name lookup. Is this a comment issue or a code issue?

Reply:
This is a case of me somehow typing the _exact opposite_ of "qualified name lookup" in this comment.